config/betterquesting/DefaultQuests/Quests/Tier1LV-AAAAAAAAAAAAAAAAAAAAAw==/PrisonGangsareOb-AAAAAAAAAAAAAAAAAAALLQ==.json

```diff
@@ -29,7 +29,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Want lots of cobble, stone, or obsidian? This is the machine for you!\n\nCobble: Water and lava on 2 of the sides.\nStone: Water on a side, lava on top.\nObsidian: Either of the above, and redstone in the machine.\n\nYou of course need to power it. If you don\u0027t want to spend power on cobble, do it the ExU way with the WIU.\n\n§3You need to toggle the ghost circuit to circuit 1 to get obsidian.\n"
+      "desc:8": "Want lots of cobble, stone, or obsidian? This is the machine for you!\n\nCobble: Water and lava on 2 of the sides.\nStone: Water on a side, lava on top.\nObsidian: Either of the above, and redstone in the machine.\n\nYou of course need to power it. If you don\u0027t want to spend power on cobble, do it the ExU way with the WIU.\n\nYou need to toggle the ghost circuit to Circuit-1 to get obsidian.\n"
```

Comment:
"Circuit-1" never seen that notation before. better just leave it

Reply:
Fixed in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/13426/commits/a748778e1681837baf3e46649516711b26215cec